static/app/components/events/autofix/autofixRootCause.tsx

```diff
@@ -123,6 +123,50 @@ function useSelectCause({groupId, runId}: {groupId: string; runId: string}) {
   });
 }
 
+function getLinesToHighlight(suggestedFix: AutofixRootCauseCodeContext): number[] {
+  function getBacktickSubstrings(input: string): Set<string> {
+    // Regular expression to match substrings wrapped in backticks
+    const regex = /`([^`]+)`/g;
+    const result = new Set<string>();
+    let match: RegExpExecArray | null;
+    do {
+      match = regex.exec(input);
+      if (match) {
+        result.add(match[1]);
+      }
+    } while (match);
+    return result;
+  }
+
+  function findLinesWithSubstrings(
+    input: string | undefined,
+    substrings: Set<string>
+  ): number[] {
+    if (!input) {
+      return [];
+    }
+    const lines = input.split('\n');
+    const result: number[] = [];
+
+    lines.forEach((line, index) => {
+      for (const substring of substrings) {
+        if (line.includes(substring)) {
+          result.push(index + 1); // line numbers are 1-based
+          break;
+        }
+      }
+    });
+
+    return result;
+  }
+
+  const lineNumbersToHighlight = findLinesWithSubstrings(
+    suggestedFix.snippet?.snippet,
+    getBacktickSubstrings(suggestedFix.description)
```

Comment:
UX wise, would it make more sense to highlight a specific line that is causing the issue rather than highlighting the lines from the description's backticks?

Reply:
Done in next commit